docs/versions.rst

```diff
@@ -114,6 +114,18 @@ If you want to share your docs temporarily, see :doc:`/commercial/sharing`.
 In addition, if you want other users to view the build page of your public versions,
 you'll neeed to the set the :doc:`privacy level of your project </commercial/privacy-level>` to public.
 
+Logging out
+'''''''''''
+
+When you log in to a documentation site, you will be logged in until close your browser.
+To log out, click on the :guilabel:`Log out` link in your documentation's flyout menu.
+This is usually located in the bottom right or bottom left, depending on the theme design.
+This will log you out from the current domain,
+but not end any other session that you have active. 
```

Comment:
Doesn't this log you out from the dashboard as well?

Reply:
Nope, but we redirect to the dashboard after logging out from the domain

Reply:
Do we redirect to the Dashboard Homepage or the Dashboard Logout page?

Reply:
I think we are still redirecting to the logout page, I think we should redirect to the home page TBH

Reply:
Agreed, that seems cleaner 👍